run_tests.sh

```diff
@@ -14,55 +14,32 @@
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #
 
+[ -n "$PYTHON" ] || PYTHON=python2
```

Comment:
defaults to python2?

Reply:
This was written a while back... Should python2 support just be dropped now?

Reply:
I looked over this patch set, and it still seems fine. The changes are minimal in the sense that python2 is still used by default. If you want, I can add another commit on top that either switches to python3 by default or completely drops python2 support.